tests/pebble.rs

```diff
@@ -557,6 +562,7 @@ struct EnvironmentConfig {
     dns_port: u16,
     challtestsrv_port: u16,
     eab_key: Option<ExternalAccountKey>,
+    reuse_all_authz: bool,
```

Comment:
How about we make this a more direct match to the Pebble config:

```rust
/// Percentage of authorizations that can be reused [maybe there's upstream docs that describe this better?]
authz_reuse: u8,
```

Reply:
Done and linked to [the upstream docs](https://github.com/letsencrypt/pebble?tab=readme-ov-file#valid-authorization-reuse).